service/host.go

```diff
@@ -323,29 +304,34 @@ func (s *host) Stop(reason string, exitCode int) error {
 	return err
 }
 
-func (s *host) startModules() map[Module]error {
-	results := map[Module]error{}
+func (s *host) startModules() []error {
```

Comment:
No idea, but this PR is just addressing the external interface, I think we can iterate on implementation some other time.